### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by lhack-jpeg**
add view page for error

### CODE REVIEW

**Review by Gotmished — APPROVED** [review]
Seems fine to me - I wouldn't know either way, haha!

File: dota_match_scheduler/dota_match_scheduler/models.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+Base = declarative_base()
+
+
+class MyEnum(enum.Enum):
+    one = "Bo1"
+    two = "Bo2"
+    three = "Bo3"
+    four = "Bo5"
```
Comment (RubberizedDuck) [inline dota_match_scheduler/dota_match_scheduler/models.py:25]: Just questioning this line of code, is it meant to be Bo4 or is Bo5 okay?

Comment (lhack-jpeg) [reply dota_match_scheduler/dota_match_scheduler/models.py:25]: Bo4 means you can have a draw, Bo5 is usually reserved for GF matches

File: tech_foul_app/routes/index.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-const router = express.Router();
+const express = require('express')
+const controllers = require('../controllers')
+const wrapAsync = require('../utilities/wrapAsync')
+const router = express.Router()
 
-router.route('/').get(controllers.getAllMatches);
-router.route('/:match_id').get(controllers.match_detail);
+router.route('/').get(wrapAsync(controllers.getAllMatches))
+router.route('/:match_id').get(wrapAsync(controllers.match_detail))
```
Comment (Gotmished) [inline tech_foul_app/routes/index.js:8]: All seems fine to me, but what does the addition of wrapAsync do?

File: tech_foul_app/update.js [hunk 3]
```diff
@@ -31,10 +31,9 @@ mongoose
...
       }]
-    });
+    })
     matchone.save(function (error, document) {
-      if (error) console.error(error);
-      console.log(document);
-    });
+      if (error) console.error(error)
+      console.log(document)
+    })
```
Comment (Gotmished) [inline tech_foul_app/update.js:38]: You can delete this file, as I was just using it to populate the MongoDB with dummy data.

File: tech_foul_app/routes/index.js (hunk 2 above)
Comment (lhack-jpeg) [reply tech_foul_app/routes/index.js:8]: wrap async uses a call back function to help handle any errors we get

Comment (Gotmished) [reply tech_foul_app/routes/index.js:8]: Ah, don't worry! I saw it further down.

File: tech_foul_app/update.js (hunk 3 above)
Comment (lhack-jpeg) [reply tech_foul_app/update.js:38]: We'll do a pruning after the pull request

File: dota_match_scheduler/dota_match_scheduler/models.py (hunk 1 above)
Comment (RubberizedDuck) [reply dota_match_scheduler/dota_match_scheduler/models.py:25]: Amazing! All looks good to me then :)

**Review by RubberizedDuck — COMMENTED** [review]
Everything seems to be fine on my end :)

**Comment by RubberizedDuck** [comment]
Everything seems to be fine on my end :)

**Pull request closed without merging**